hack/feature-deploy.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
 
+export KUSTOMIZE_DIR="${KUSTOMIZE_DIR:-/tmp}"
+export KUSTOMIZE_BIN=$KUSTOMIZE_DIR/kustomize
+
+# Function: download the latest Kustomize binary to /tmp
+function get_kustomize_binary (){
+    pushd /tmp
+    curl -s "https://raw.githubusercontent.com/\
+kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+    popd
+}
+
```

Comment:
this whole function should move after the common part

Reply:
Hi @yuvalk!
If I move function `get_kustomize_binary` after the common part, I will have to encapsulate the common part into the `main` function. And even then main should be the bottom most function, according to this: [Google's shell style guide](https://google.github.io/styleguide/shell.xml).

Reply:
not sure what do you mean by `main` function, we dont have have a `main` in this script.

common is just [this](https://github.com/openshift-kni/cnf-features-deploy/blob/master/hack/common.sh) script 
so I'm not sure what's the problem with moving the function adter the line that source it
the added value is the `set -e` which I dont think we should have any code without it
and even in special cases, it's better to explicitly `set +e` (like we are doing elsewhere [here for example](https://github.com/openshift-kni/cnf-features-deploy/blob/master/hack/feature-deploy.sh#L38)

Reply:
I merely meant that you should move the function definition after the line that source the common.sh

Reply:
Ah sorry I misunderstood it. Now I got  what you mean by the common part.